README.md

```diff
@@ -1,2 +1,104 @@
-# rust-vmm-container
-Container with all dependencies required for running rust-vmm crates integration tests. 
+# rust-vmm-dev-container
+
+**rust-vmm-dev** is a container with all dependencies used for running
+rust-vmm integration tests.
+
+The container is available on Docker Hub and has support for x86_64 and
+aarch64 platforms.
+Pull the latest version by running:
+
+```bash
+docker pull fandree/rust-vmm-dev
+```
+
+Depending on which platform you're running the command from, docker will pull
+either `rust-vmm-dev:aarch64` or `rust-vmm-dev:x86_64`.
+
+For now rust is installed only for the root user.
+
+### Using the container
+
+The container is currently used for running the integration tests for the
+[kvm-ioctls](https://github.com/rust-vmm/kvm-ioctls) crate.
+
+Example of running cargo build on the kvm-ioctls crate:
+
+```bash
+> git clone git@github.com:rust-vmm/kvm-ioctls.git
+> cd kvm-ioctls/
+> docker run --volume $(pwd):/kvm-ioctls \
```

Comment:
What do you mean by helper script?

Reply:
Something like [firecracker's devtool](https://github.com/firecracker-microvm/firecracker/blob/master/tools/devtool)

Reply:
I was afraid you'll say that :))

If we have a helper script it would only be used by people testing their code on their local machines. That is because with buildkite the command will look different making the script unusable. This is an example of running cargo build with the container on Buildkite:

```
  - label: "build-gnu-x86"
    commands:
     - cargo build --release
    retry:
      automatic: false
    agents:
      platform: x86_64.metal
    plugins:
      - docker#v3.0.1:
          image: "fandree/rust-vmm-dev"
          always-pull: true
```
Also, the situation is a bit different with rust-vmm components because we do not have both the container and the code in the same repository so we cannot do fancy things like attaching the volume (as in the crate code) with a helper script because we do not know where the volume is. Maybe we can hack something together to make it work, but I don't think it adds a lot of value to be worth the trouble.

Reply:
Aah didn't take the Buildkite magic into consideration (or the decoupling of crates & container), I agree it's overkill and unnecessary to add (the same) helpers in every repo